src/enclave/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ if (OE_CRYPTO_LIB STREQUAL "openssl_3")
 endif ()
 
 add_executable(ortoa-enc 
-    common/ecalls.cpp
+    ecalls.cpp
     ../crypto/encryption_engine.cpp
```

Comment:
(can be left for later PR) If we're using path variables like `${CMAKE_CURRENT_BINARY_DIR}, I propose we keep it consistent and try not to use relative paths as well

Reply:
yep future refactor indeed. Thus far I've just been getting things to work (compile) rather than doing them properly